corehq/apps/accounting/static/accounting/js/credits.js

```diff
@@ -37,24 +37,38 @@ hqDefine('accounting/js/credits', [
         self.products = products;
         self.features = features;
         self.paymentHandler = paymentHandler;
-        self.general_credit = ko.observable(generalCreditItem(paymentHandler));
+        self.cc_general_credit = ko.observable(ccGeneralCreditItem(paymentHandler));
```

Comment:
also if we are renaming this variable...can we camel case the variable since it's in javascript? we seem to have mixed variable naming schemes here

Reply:
This is a good call and will fix today. I realized I had mixed a few of these up when I put up this PR, but didn't have the brainspace to address it right then.

Reply:
https://github.com/dimagi/commcare-hq/pull/36064/commits/4a6076ed8793992dfa1ffca5833eba97eeb02ebc